src/ui/i18n/en-US.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
should not need to have an `en` and `en-US` — need to be able to handle fallback. I thought the accept language parser you are using would do some fallback mapping.   `en-US` should fall back to `en` - perhaps this was before you had default language support though.